src/generators/bendable-amogus/BendableAmogus.res

```diff
@@ -0,0 +1,239 @@
+let requireImage = fileName => Generator.requireImage("./images/" ++ fileName)
+let requireTexture = fileName => Generator.requireImage("./textures/" ++ fileName)
+
+let id = "bendableamogusgenerator"
+
+let name = "Bendable Amogus generator"
```

Comment:
Let's rename this generator:

Directory name: `among-us-bendable`

File name: `AmongUsBendableGenerator.res`

id: `among-us-bendable`

name: `Among Us Bendable`